### ISSUE TITLE
handleDisconnect is not triggered when using WsAdapter

### ISSUE THREAD

**Issue by zelishe** [nestjs/nest#2226]
## I'm submitting a...

<pre><code>
[ ] Regression 
[X] Bug report
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead post your question on Stack Overflow.
</code></pre>

## Current behavior
If you are using WsAdapter, then handleDisconnect in the WebSocketGateway is never called upon the client disconnect.

## Expected behavior
handleDisconnect must be triggered, when client websocket is closing the connection.

## Minimal reproduction of the problem with instructions
**main.ts**
```
async function bootstrap() {

  const app = await NestFactory.create<NestExpressApplication>(AppModule);

  await app.get(BootService).onApplicationInit();

  app.useWebSocketAdapter(new WsAdapter(app));

  await app.listen(3000);

}

```

**websocket.gateway.ts**
```
@WebSocketGateway(4242)
export class WebsocketGateway implements OnGatewayDisconnect {

  constructor() {}

  handleDisconnect(frontendWebsocketClient: IFrontendWebsocketClient): any {
    // This event is not being triggered when client disconnects
    console.log('handleDisconnect() triggered');
  }

}
```

## Environment

```
Nest version: 6.2.0
```

**Comment by zelishe** [comment]
This happens because platform-ws' WsAdapter produces CLOSE_EVENT
```
const close$ = fromEvent(client, CLOSE_EVENT).pipe(share(), first());
```
But websocket's AbstractWsAdapter is waiting on DISCONNECT_EVENT
```
public bindClientDisconnect(client: TClient, callback: Function) {
    client.on(DISCONNECT_EVENT, callback);
}
```
It works fine with platform-socket.io (DISCONNECT_EVENT is fired there), so, I suppose that WsAdapter in platform-ws must do the same, am I right?

**Comment by kamilmysliwiec** [comment]
Fixed in 6.2.1 :) Thanks for reporting
